docs/CHANGES.rst

```diff
@@ -10,13 +10,17 @@ Breaking changes:
 
 New features:
 
-- *add item here*
+- Allow to only get local roles for groups by passing inherit=False to groups.get_roles()
```

Comment:
I would rephrase this as follows:
"Passing inherit=False to groups.get_roles() will only get local roles for the group."